lib/xdrgen/generators/rust.rb

```diff
@@ -388,18 +388,20 @@ def render_source_comment(out, defn)
 
       def render_struct(out, struct)
         out.puts "#[derive(Clone, Debug, Hash, PartialEq, Eq, PartialOrd, Ord)]"
+        out.puts %{#[cfg_eval::cfg_eval]}
```

Comment:
`cfg_eval` is added on each type that uses serde_as field attributes, to add support for using cfg_attr on those field attributes.